### PR TITLE
fix(general-petition): General Petition Template, vol.2

### PR DESCRIPTION
**Pull request by albinagu**
## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/feature-flags/src/lib/features.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ export enum Features {
   noDebtCertificate = 'applicationTemplateNoDebtCertificateEnabled',
   drivingInstructorRegistrations = 'isDrivingInstructorRegistrationsEnabled',
   drivingSchoolConfirmations = 'isDrivingSchoolConfirmationsEnabled',
+  generaPetition = 'isGeneralPetitionEnabled',
```
Comment (saevarma) [inline libs/feature-flags/src/lib/features.ts:22]: FYI typo in `generalPetition`

**Review by saevarma — APPROVED** [review]
Core files LGTM, just one typo.

**Review by alexdiljar — APPROVED** [review]
lgtm

**Review by Toti91 — APPROVED** [review]
Code owned looks good

**Pull request merged**